ompi/mca/coll/hcoll/coll_hcoll_ops.c

```diff
@@ -121,7 +121,8 @@ int mca_coll_hcoll_allgatherv(const void *sbuf, int scount,
     mca_coll_hcoll_module_t *hcoll_module = (mca_coll_hcoll_module_t*)module;
     stype = ompi_dtype_2_hcoll_dtype(sdtype, NO_DERIVED);
     rtype = ompi_dtype_2_hcoll_dtype(rdtype, NO_DERIVED);
-    if (OPAL_UNLIKELY(HCOL_DTE_IS_ZERO(stype) || HCOL_DTE_IS_ZERO(rtype))) {
+    if (OPAL_UNLIKELY(HCOL_DTE_IS_ZERO(stype) || HCOL_DTE_IS_ZERO(rtype)
+				              || ompi_count_array_is_64bit(rcount))) {
```

Comment:
This is probably something that someone from @janjust may want to look at: should hcoll disqualify itself if the count argument is 64bit?

Reply:
@janjust Ping